pluton/harness/run.go

```diff
@@ -45,6 +47,38 @@ func RunSuite(pattern string) {
 	}
 	suite := harness.NewSuite(opts, tests)
 
+	// setup a node for in cluster services not tied to individual test life-cycles
+	switch Opts.CloudPlatform {
+	case "gce":
+		var bastionDir = filepath.Join(Opts.OutputDir, "bastion")
+
+		if err := os.MkdirAll(bastionDir, 0777); err != nil {
+			fmt.Printf("setting up bastion cluster: %v\n", err)
+			os.Exit(1)
+		}
+
+		// override machine size for bastion
+		savedVal := Opts.GCEOptions.MachineType
+		Opts.GCEOptions.MachineType = "n1-standard-4"
+
+		cloud, err := gcloud.NewCluster(&Opts.GCEOptions, bastionDir)
```

Comment:
Since the cluster is discarded and only the machine is kept there isn't anything to call cluster.Destroy. I think the only thing that leaks as a result is the ssh agent socket which isn't a huge deal but if you can it would be good to destroy the cluster too.